app/src/main/java/org/simple/clinic/overdue/AppointmentRepository.kt

```diff
@@ -0,0 +1,60 @@
+package org.simple.clinic.overdue
+
+import io.reactivex.Completable
+import io.reactivex.Single
+import org.simple.clinic.patient.SyncStatus
+import org.simple.clinic.patient.canBeOverriddenByServerCopy
+import org.simple.clinic.sync.SynceableRepository
+import java.util.UUID
+import javax.inject.Inject
+
+class AppointmentRepository @Inject constructor(val dao: Appointment.RoomDao) : SynceableRepository<Appointment, AppointmentPayload> {
+
+  fun save(vararg schedules: Appointment): Completable {
```

Comment:
Since we are using `List` everywhere, can we use lists here as well instead of `vararg`?

Reply:
Hrm, I chose to use vararg because I want to be able to send both one item or multiple items. I could also just default to `List`

Reply:
Right, I just want to have the same conventions everywhere.